### ISSUE TITLE
Motor control: motors not being updated often enough

### ISSUE THREAD

**Issue by WubbaDuck** [Team74/FRC_2016_Python_Stronghold#28]
@Team74/programmers 

The motors constantly output an error saying that they are not being updated often enough. This error may be due to the watchdog timer not being fed. See [https://wpilib.screenstepslive.com/s/3120/m/8559/l/180753-troubleshooting](url) for more info 

> Motors pulse on and off and/or the message "Output not updated often enough": Check that the motor safety is either disabled or that the motors speeds are updated periodically. The problem is that the watchdog isn't being "fed" and shuts off the motor signal every 100ms. This will be accompanied by a message that says "Output not updated often enough".

The [watchdog timer](https://en.wikipedia.org/wiki/Watchdog_timer) is an internal timer that will reset the hardware in the event that the code hangs or takes too long to complete

An example of how to get the watchdog timer and feed it can be found here [https://github.com/jacob9706/FRC_Python_Scheduler/blob/master/robot.py](url)

And another example: [https://github.com/robertlong/robotpy-bootstrap/blob/master/src/robot.py](url)

Cute puppy tax for watchdog timer ![Puppy Tax](https://s-media-cache-ak0.pinimg.com/originals/66/fb/12/66fb120e0d4a21a624b9cf519534e231.gif)

**Comment by WillHescott** [comment]
Are you sure it isn't motorsafey? We haven't tried disabling it yet

**Comment by WubbaDuck** [comment]
The motor safety functionality is in there to make sure that our robot doesn't go nuts if we loose communication to the robot. Knowing how FIRST things go this is guaranteed to happen and it would be irresponsible to risk that kind of robot behavior. Also, we did try disabling it with no change in behavior. I looked at the motorsafety code and the watchdog timer code. Both have the ability to throw the warning that we are seeing. If the issue is the watchdog timer then we will have bigger issues come competition time than just slow motors. That could lead to a full reset mid match.
